src/db-service.ts

```diff
@@ -64,6 +68,18 @@ export class ValidationService{
       .then(({ rows }) => rows[0].id);
   }
 
+  async getAllStatus(transactionHandler: TrxHandler): Promise<Status[]> {
```

Comment:
Oh I see now. Then `getAllItemValidationsAndItemValidationReviewsStatuses`? Actually it would be clearer to have 2 different calls and the front merge them. Let me know what you think.

Reply:
Ok, then I can go with 2 different calls.